### DISCUSSION TITLE
It's not work for mysql database

### DISCUSSION THREAD

**Discussion by jieTan2018** [mmRoshani/sequelize-typescript-migration#12] [Q&A]
![image](https://user-images.githubusercontent.com/37867079/227711775-9d0f94b3-4c12-4b09-b5f9-46ea37d8e6ee.png)
The automatically generated database table name is incorrect. How to resolve it? @mmRoshani

**Comment by mmRoshani** [comment]
@jieTan2018 

Hi. didn't [this](https://github.com/mmRoshani/sequelize-typescript-migration/discussions/8) fixed the issue?
